src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -135,10 +136,11 @@
     color: #ffffff;
   }
   
-  .outlines .isComparison:hover, .isComparison{
+  .outlines .buttonComparison:hover, .isComparison{
```

Comment:
po prostu na button daj ten hover, nie na określoną klasę

Reply:
ale button ma tag "a", o to chodzi?

Reply:
hmm faktycznie, to może dodaj jeszcze mu jakąś ogólną klasę, np. `btn`